src/open_inwoner/accounts/views/auth.py

```diff
@@ -133,8 +140,30 @@ def get_success_url(self):
         return super().get_success_url()
 
 
+class BlockEenmanszaakLoginMixin:
+    def get(self, request):
+        response = super().get(request)
+
+        openzaak_config = OpenZaakConfig.get_solo()
+        openklant_config = OpenKlantConfig.get_solo()
+        if (
+            hasattr(request.user, "rsin")
+            and not request.user.rsin
```

Comment:
Looks like `getattr(request.user, "rsin", None)`

Reply:
`request.user` can be AnonymousUser (in case OIDC gives errors) and in that case I want the OIDCCallbackView to handle those errors